test/unit/bacnet-services.spec.js

```diff
@@ -233,8 +233,32 @@ describe('bacstack - Services layer', function() {
           propertyIdentifier: 81
         },
         valueList: [
-          {type: 7, value: '', len: 2},
-          {type: 7, value: 'Test1234$', len: 12}
+          {type: 7, value: '', len: 2, encoding: 0},
+          {type: 7, value: 'Test1234$äöü', len: 18, encoding: 0}
+        ]
+      });
+    });
+
+    it('should successfully encode and decode a character-string value with ISO-8859-1 encoding', function() {
+      var buffer = utils.getBuffer();
+      baServices.EncodeReadPropertyAcknowledge(buffer, {type: 8, instance: 40000}, 81, 0xFFFFFFFF, [
+        {Tag: 7, Value: '', Encoding: 5},
+        {Tag: 7, Value: 'Test1234$äöü', Encoding: 5} // Encoding: 5 => ISO-8859-1
```

Comment:
Not really a review finding, but more of a question: What do you think if we add baEnum to the tests. E.g:

```
{Tag: 7, Value: 'Test1234$äöü', Encoding: baEnum.BacnetCharacterStringEncodings.CHARACTER_ISO8859_1}
```